### PR TITLE
Copied Blogs from smp-spec.io to meshery.io/blog

### LINKED ISSUE #1395: Copy blogs from smp-spec.io to meshery.io/blog
#### Current State of the Meshery.io site

The blogs on the meshery.io site can be found at https://meshery.io/blog.
Head over to https://smp-spec.io/ and locate the BLOG section on the homepage

#### Desired State of the Meshery.io site

Copy the blogs from https://smp-spec.io/ to https://meshery.io/blog while maintaining the original date and author's name.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by akhilender-bongirwar**
**Description**

This commit implements the solution to the issue regarding the migration of blogs from https://smp-spec.io/ to https://meshery.io/blog. The blogs have been successfully copied while preserving the original publication date and author's name.

Changes Made:

- Copied blogs from https://smp-spec.io/ to https://meshery.io/blog.
- Ensured that the original publication date and author's name are maintained.

This PR fixes #1395 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@akhilender-bongirwar 
Let's discuss it on the websites call.
Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

**Review by Mohith234 — CHANGES_REQUESTED** [review]
- There are some places where an unordered list is used instead of an ordered list, like in `Meshmark Explained` and `Getting Started with SMP Analysis`. Could you check them?
- Also, reduce the space between the image and the text below it. They are set wide apart on the home page [ `/blog` page ]
- On the `Pipelining Service Mesh Specifications` the code snippet lacks indentations in the code which are highly important. Please fix them.

The rest looks good to me @akhilender-bongirwar

**Comment by akhilender-bongirwar** [comment]
Hi @Mohith234 , I have reviewed and made the required change. Please check : ) ... Thank you

**Comment by Mohith234** [comment]
Still, the indentation in the code didn't get fixed [ 3rd point mentioned above ] 

// @akhilender-bongirwar

**Comment by akhilender-bongirwar** [comment]
> Still, the indentation in the code didn't get fixed [ 3rd point mentioned above ]
> 
> // @akhilender-bongirwar

Sorry @Mohith234 , didn't notice that the prettier in my vscode was doing it. I will fix that soon.... : )..

**Comment by akhilender-bongirwar** [comment]
Hey @Mohith234 ,  Could you please check the changes.. : )...

**Review by Mohith234 — CHANGES_REQUESTED** [review]
- To make the text look consistent, change the text color to default with the other text. Attached a screenshot of the `Meshmark Explained` blog post. The same goes for `Analyzing with SMP` 

image

- The headings inside the `Analyzing with SMP` blog can be made larger. Maybe h2 or h3 would look better IMO. What do you think? @akhilender-bongirwar 

- In `Pipelining Service mesh specifications` this is a wrong syntax remove the space before the `name`
image

**Comment by Mohith234** [comment]
Sorry for the late reply. I pointed out some of the things to improve above, please look into them and discuss your opinion on incorporating those changes. 

Also, is there any other way of adding indentations other than adding `&nbsp` for the indentations part?

// @akhilender-bongirwar

**Comment by akhilender-bongirwar** [comment]
Hey @Mohith234 no problem. I have just seen the reviews

- The text being inconsistent in light mode maybe has occured due to other pull request #1407 . I will fix that soon. 
- Yeah, the headings in `Analysing with SMP`  with h3 will look good IMO.
- I have copied the exact indentations as it was there in the `https://smp-spec.io/blog/spec/pipelining-service-mesh-specifications`
The screenshot of the original indentations for the Pipelining Service Mesh Specifications
Screenshot 2023-09-10 004059
The original one also has an extra space for `name`.


- Only `&nbsp;` is working for the indentations part or I can reduce the &nbsp; by `&ensp; = two &nbsp;`
`&emsp; = four &nbsp;` . If there is any other more feasible alternative please let me know  : ).

**Comment by Mohith234** [comment]
- About the 3rd point I checked the code using YAML validator so removing the space gives valid code so..
- About reducing `&nbsp` 's , go ahead it reduces a bit of excessive code.

**Review by Mohith234 — COMMENTED** [review]
All good to go, made some suggestions take a look into them.

P.S Great work on the code snippets

File: collections/_posts/2016-07-20-the-process-for-direct-sales.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ categories:
 image: /assets/images/posts/2019-05-21-a-standard-interface-for-service-meshes/smi-logo.png
 # permalink: /blog/a-standard-interface-for-service-meshes
 ---
-<div class="row" style="width:220px;float:right;padding:20px;text-align:center;">
+<div class="row" style="width:220px;float:right;padding:5px;text-align:center;">
```
Comment (Mohith234) [inline collections/_posts/2016-07-20-the-process-for-direct-sales.md:13]: - This blog is not listed I guess
```suggestion
<div class="row" style="width:220px;float:right;padding:20px;text-align:center;">
```

File: collections/_posts/2023-06-16-changing-meshery-release-channels.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ categories:
 permalink: /blog/changing-meshery-release-channels
 ---
 
-<div style="text-align:center;padding:20px">
+<div style="text-align:center;padding:5px">
```
Comment (Mohith234) [inline collections/_posts/2023-06-16-changing-meshery-release-channels.md:14]:
```suggestion
<div style="text-align:center;padding:20px">
```

File: collections/_posts/2023-01-09-meshery-system-provider.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ categories:
 permalink: /blog/mesheryctl-system-provider
 ---
 
-<div style="text-align:center;padding:20px">
+<div style="text-align:center;padding:5px">
```
Comment (Mohith234) [inline collections/_posts/2023-01-09-meshery-system-provider.md:13]:
```suggestion
<div style="text-align:center;padding:20px">
```

File: collections/_posts/2021-04-26-validating-smi-conformance-with-meshery.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ categories:
 # permalink: /blog/validating-smi-conformance-with-meshery
 ---
 
-<div class="row" style="width:220px;float:left;padding:20px;text-align:center;">
+<div class="row" style="width:220px;float:left;padding:5px;text-align:center;">
```
Comment (Mohith234) [inline collections/_posts/2021-04-26-validating-smi-conformance-with-meshery.md:12]:
```suggestion
<div class="row" style="width:220px;float:left;padding:20px;text-align:center;">
```

**Comment by akhilender-bongirwar** [comment]
Hey @Mohith234, could you please review the pr as I have implemented all the suggested changes as you said.

**Review by Mohith234 — APPROVED** [review]
Awesome work done @akhilender-bongirwar 

Looks good to me 🚀

**Comment by leecalcote** [comment]
Merge conflict...

File: collections/_posts/2020-05-21-meshmark-explained.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+     * Negative: how concerned am I that this service mesh feature will fail?
+
+
+MeshMark functions as a service mesh performance index (a scale) to provide people the ability to weigh the value of their service mesh versus the overhead of their service mesh and assess whether they are getting out of the mesh what they are “paying” for in it.
+
+The scoring system ranges from 0 to 100.
+<div style="display: flex; align-items: center; justify-content: space-around">
+    <div>
+        <p>
+            Another aspect here is the need for distributed, multi-mesh and workload performance management. Distributed load testing offers insight into system behaviors that arguably more accurately represent real-world behaviors of services under load as that load comes from any number of sources. This aim of enhancing the current load generation and analysis techniques to include distributed load testing is hoped to be achieved by the <a href="https://layer5.io/projects/service-mesh-distributed-performance-management">Distributed Performance Management of Service Meshes</a> project, by analyzing and working hand in hand with Nighthawk, a versatile HTTP load testing tool.
```
Comment (vishalvivekm) [inline collections/_posts/2020-05-21-meshmark-explained.md:38]:
```suggestion
            Another aspect here is the need for distributed, multi-mesh and workload performance management. Distributed load testing offers insight into system behaviors that arguably more accurately represent real-world behaviors of services under load as that load comes from any number of sources. This aim of enhancing the current load generation and analysis techniques to include distributed load testing is hoped to be achieved by the <a href="https://layer5.io/projects/service-mesh-performance">Distributed Performance Management of Service Meshes</a> project, by analyzing and working hand in hand with Nighthawk, a versatile HTTP load testing tool.
```

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
@leecalcote 
The link was broken updated it with : https://layer5.io/projects/service-mesh-performance

File: collections/_posts/2022-01-26-pipelining-service-mesh-specifications.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+    </a>
+</div>
+
+You can download a token from Meshery and add it as a GitHub secret (in the example above, the secret is MESHERY_PROVIDER_TOKEN). After the test is run, you can view the results from the Service Mesh Interface dashboard in Meshery UI.
+
+<div style="text-align:center;margin-bottom:2rem;width:100%">
+<img width="100%" src="../../assets/images/posts/2022-01-26-pipelining-service-mesh-specifications/smi-conformance-result.png" alt="smi conformance dashboard">
+</div>
+
+Participating service mesh projects can also <a href="https://docs.meshery.io/functionality/service-mesh-interface#reporting-conformance">automatically report their conformance test results</a> to the <a href="https://meshery.io/service-mesh-interface">SMI Conformance dashboard</a>
```
Comment (vishalvivekm) [inline collections/_posts/2022-01-26-pipelining-service-mesh-specifications.md:167]:
```suggestion
Participating service mesh projects can also <a href="https://docs.meshery.io/tasks/service-mesh-interface#reporting-conformance">automatically report their conformance test results</a> to the <a href="https://meshery.io/service-mesh-interface">SMI Conformance dashboard</a>
```

File: _sass/layout.scss [hunk 7]
```diff
@@ -378,7 +378,7 @@ hr {
     line-height: 25px;
     width: 90%;
     font-size: 15px;
-    white-space: pre-line;
```
Comment (vishalvivekm) [inline _sass/layout.scss:381]: This causes:
Screenshot from 2023-10-08 03-00-33

File: _sass/layout.scss [hunk 8]
```diff
@@ -378,7 +378,7 @@ hr {
...
+    white-space: pre-wrap;
```
Comment (vishalvivekm) [inline _sass/layout.scss:381]:
```suggestion
    white-space: pre-line;
```

**Review by vishalvivekm — APPROVED** [review]

**Pull request merged**